cmd/integration_test.go

```diff
@@ -64,9 +66,35 @@ func conformResult(t *testing.T, test *TestCase, resp *http.Response, err error)
 		if !a.NoError(err) {
 			return
 		}
-		a.Contains(string(body), "egress proxying denied to host")
+		a.Contains(string(body), "egress proxying denied")
 		a.Contains(string(body), "moar ctx")
 	}
+
+	var entries []*logrus.Entry
+	for _, entry := range logs {
+		if entry.Level < logrus.WarnLevel {
+			a.Failf("unexpected log line more severe than Warn", "%v", entry)
+		} else if entry.Level < logrus.DebugLevel {
+			entries = append(entries, entry)
+		}
+	}
+	a.Equalf(1, len(entries), "Expected only a single INFO/WARN log line per request, got: %v", entries)
```

Comment:
Is it going overboard asserting that there's a single line per request? We could relax this to require only a single "proxy_response" line per request.

Reply:
I'm okay with a single "proxy_response" line per request!